src/main/java/Duke.java

```diff
@@ -1,10 +1,53 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        String dukeLogo =  " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String convStart = dukeLogo + "Hello! I'm Duke\n" + "What can I do for you?";
+        String convEnd= "Bye. Hope to see you again soon!";
+
+        ListDuke list = new ListDuke();
+
+        printOutput(convStart);
+
+        String lineInput;
+        Scanner in = new Scanner(System.in);
+        while (true) {
+            lineInput = in.nextLine();
+            if (lineInput.equals("bye")) {
+                break;
+            } else if (lineInput.equals("list")) {
+                printOutput(list.getInputLists());
+            } else if (lineInput.startsWith("mark ")) {
+                int itemNo = Integer.parseInt(lineInput.split(" ")[1]);
+                list.markCompleted(itemNo, true);
+                String message = "Nice! I've marked this task as done: \n"
+                        + list.getItemFromList(itemNo);
```

Comment:
Good use of linebreak for a long line, and the break is before the `+` operator. Great job!